src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/service/SendDecisionAndReasonsPdfService.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.service;
```

Comment:
This should probably live in the infrastructure package following our DDD structure.